extension/mysql_to_doris/conf/tables

```diff
@@ -0,0 +1,44 @@
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+####################################################################
+# The tables is used to define tables
+####################################################################
```

Comment:
This file is not a shell script. When the user_define_tables.sh script is used to read ./conf/tables, the above contents will be read, resulting in the wrong table name being obtained and the script execution failing. These contents need to be removed.

Reply:
delete all content